code/game/gamemodes/miniantags/morph/morph.dm

```diff
@@ -76,24 +76,12 @@
 	morphed = 1
 	form = target
 
-	//anim(loc,src,'icons/mob/mob.dmi',,"morph",,src.dir) No effect better than shit effect
-
-	//Todo : update to .appearance once 508 hits
-	if(ishuman(target))
-		var/mob/living/carbon/human/H = target
-		overlays = H.get_overlays_copy(list(L_HAND_LAYER,R_HAND_LAYER))
-	else
-		overlays = target.overlays.Copy()
-
+	appearance = target.appearance
+	transform = matrix()
+	pixel_y = 0
+	pixel_x = 0
```

Comment:
The morph's initial is always 0 for pixel_x and pixel_y, didn't think it was needed.